src/guide/jbs-jdk-bug-system.md

```diff
@@ -214,6 +214,7 @@ When triaging an issue, first give it a general review.
 
 1. If the issue is a duplicate, close it as such.
 1. If the issue belongs to a different area (it was filed in libraries, but it's an HotSpot issue), transfer it to the correct component/subcomponent making sure that the state remains [New]{.jbs-value}.
+   * Please note that issues in the [hotspot]{.jbs-value} and [security-libs]{.jbs-value} components must have a [Subcomponent]{.jbs-field} set as well.
```

Comment:
Are these the only components which require a subcomponent?  I know that client-libs and core-libs are quite large (https://bugs.openjdk.org/projects/JDK?selectedItem=com.atlassian.jira.jira-projects-plugin:components-page) so it wouldn't surprise me if there were similar recommendations there too.  Perhaps update to "issues in the hotspot, security-libs, and other components ..." to leave this possibility open without needing to survey all of the components?

Reply:
As far as I know the other components don't require a subcomponent.